.gitignore

```diff
@@ -38,3 +38,8 @@ GoldenDict.xcodeproj/
 *.opensdf
 *.suo
 *.vcxproj.user
+
+# QT Creator files
+goldendict.files
+goldendict.creator*
+goldendict.config
```

Comment:
I don't understand why your Qt Creator needs these files at all, and therefore why they should be added or ignored. I recommend dropping the first two commits altogether.